api/api/utils/image_proxy/__init__.py

```diff
@@ -106,26 +141,36 @@ def get(
     )
 
     try:
-        upstream_response = requests.get(
+        # todo: refactor to use aiohttp shared session
```

Comment:
Do we have an issue for this?

Reply:
Cleaned up in https://github.com/WordPress/openverse/pull/3388/commits/1d6ebd52f5759a93a7c7cae346faef8cc3726f02